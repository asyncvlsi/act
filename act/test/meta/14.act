defptype mytype(pint a, b);

template<myptype x>
defproc test(bool i[x.a]; bool o[x.b])
{
  i[0] = o[0];
}

