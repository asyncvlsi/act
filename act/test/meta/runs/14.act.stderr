ERROR: File `14.act', line 3, col 18
	Could not find specified type: myptype
